src/Cli/dotnet/commands/dotnet-workload/list/WorkloadListCommand.cs

```diff
@@ -94,26 +94,28 @@ public override int Execute()
                     var manifestInfoDict = _workloadListHelper.WorkloadResolver.GetInstalledManifests().ToDictionary(info => info.Id, StringComparer.OrdinalIgnoreCase);
 
                     InstalledWorkloadsCollection installedWorkloads = _workloadListHelper.AddInstalledVsWorkloads(installedList);
+                    var shouldPrintVersionFromInstallState = globalJsonInformation is null &&
```

Comment:
I'd suggest renaming this, I was originally confused about what it was.  I thought it was referring to the workload version in the install state, ie it would only be true if you had a pinned version in the install state.

Something like `usingWorkloadSetMode` might be clearer.

Reply:
I had some issues with this name, to be honest. usingWorkloadSetMode isn't quite right either, as you could still be using workload sets if you specify one in the global.json, but this would be false. I'm not sure what the ideal name is...

Reply:
How about `shouldShowWorkloadSetVersion`?  I think you could also set it to true if there's a workload set in global.json, because in that case we wouldn't want the manifest version column either.